client/src/views/RepoViews.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { http } from "../api/http";
+
+import styled from "styled-components";
+import logo from "../logo.svg";
+import PanelContainer from "../components/PanelContainer";
+
+function RepoViews() {
+  const [content, setData] = useState([]);
+  const [author, setAuthor] = useState("");
+  const [isLoading, setLoading] = useState(false);
+
+  async function fetchData(author) {
+    const result = await http.get(`/api/${author}`);
+    setData(result.data);
+    setLoading(false);
+  }
+
+  function onAuthorSubmit(e) {
+    if (e.charCode === 13) {
+      fetchData(author);
+    }
+  }
+
+  function onRepoSubmit() {
+    setLoading(true);
+    fetchData(author);
+  }
+
+  function renderUserInput() {
+    return (
+      <RepoInput>
+        <input
+          value={author}
+          onChange={e => {
+            setAuthor(e.target.value);
+          }}
+          onKeyPress={e => onAuthorSubmit(e)}
+        />
+        <button onClick={onRepoSubmit} disabled={!author}>
+          GO
+        </button>
+      </RepoInput>
+    );
+  }
+
+  return (
+    <div>
+      {renderUserInput()}
+      {isLoading ? (
```

Comment:
I think that here you can use the same style as @intojs said above:
```
a > b
   ? ...
   : ....
```

Reply:
Neah... I agree with using negations but the indentation not so much. React makes ternaries pretty visible in the case:
```
{isLoading ? (
    <img src={logo} className="App-logo" alt="logo" />
) : (
    <PanelContainer content={content} />
)}
```